### ISSUE TITLE
kraken2

### ISSUE THREAD

**Issue by rditter** [ubccr/software-layer#227]
Kraken2
https://github.com/DerrickWood/kraken2

**Comment by tonykew** [comment]
Test build of Kraken2 version 2.1.2 completed - more info to follow...

**Comment by tonykew** [comment]
Kraken2 version 2.1.2 has been published to the ccrsoft/2023.01 software release.

```
vortex1$ module spider kraken2
----------------------------------------------------------------------------
  kraken2: kraken2/2.1.2
----------------------------------------------------------------------------
    Description:
      Kraken is a system for assigning taxonomic labels to short DNA
      sequences, usually obtained through metagenomic studies. Previous
      attempts by other bioinformatics software to accomplish this task
      have often used sequence alignment or machine learning techniques
      that were quite slow, leading to the development of less sensitive
      but much faster abundance estimation programs. Kraken aims to achieve
      high sensitivity and high speed by utilizing exact alignments of
      k-mers and a novel classification algorithm.

    Properties:
      Bioinformatic libraries/apps

    You will need to load all module(s) on any one of the lines below before the "kraken2/2.1.2" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
vortex1$ module load gcc/11.2.0 openmpi/4.1.1 kraken2/2.1.2
vortex1$ 
```

Kraken2 includes many binaries:

```
vortex1$ ls ${EBROOTKRAKEN2}/bin
16S_gg_installation.sh       dump_table
16S_rdp_installation.sh      estimate_capacity
16S_silva_installation.sh    kraken2
add_to_library.sh            kraken2-build
build_db                     kraken2-inspect
build_gg_taxonomy.pl         kraken2lib.pm
build_kraken2_db.sh          lookup_accession_numbers
build_rdp_taxonomy.pl        lookup_accession_numbers.pl
build_silva_taxonomy.pl      make_seqid2taxid_map.pl
classify                     mask_low_complexity.sh
clean_db.sh                  rsync_from_ncbi.pl
cp_into_tempfile.pl          scan_fasta_file.pl
download_genomic_library.sh  standard_installation.sh
download_taxonomy.sh
vortex1$ 
```

e.g.

```
vortex1$ kraken2 --help
Usage: kraken2 [options] <filename(s)>

Options:
  --db NAME               Name for Kraken 2 DB
                          (default: none)
  --threads NUM           Number of threads (default: 1)
[...]
vortex1$ 

```
